### DISCUSSION TITLE
BACnet/SC

### DISCUSSION THREAD

**Discussion by mattb351** [bacnet-stack/bacnet-stack#677] [General]
I'm trying to work out if BACnet stack supports BACnet/SC but I can't seem to find any discussions on it. Does it support BACnet/SC. If not, are there any plans to implement it in the future?

**Comment by skarg** [comment]
There was considerable work to add BACnet/SC to the BACnet stack library, but pull request #423 has not been merged yet. I have refactoring the non-BACnet/SC parts of the pull request into master branch every now and then to make the pull request simpler so it can be well understood. Issue #59 discussed another BACnet/SC implementation.

**Comment by mattb351** [comment]
Thanks, I’ll take a look. 

 

--

Matt Boland

Ph: 0408351421
